AzureVM.bicep

```diff
@@ -0,0 +1,208 @@
+// The name of your Virtual Machine.
+// az deployment group create --template-file AzureVM.bicep --resource-group "test_group"
+// az bicep build --file .\AzureVM.bicep --outfile .\AzureVM.json
+param vmName string = 'mockGroundstation'
+
+// Type of authentication to use on the Virtual Machine. SSH key is recommended.
+
+@allowed([
+  'sshPublicKey'
+  'password'
+])
+param authenticationType string = 'password'
+
+// SSH Key or password for the Virtual Machine. SSH key is recommended.
+param adminPassword string
```

Comment:
For the parameters (`vmName`, `authenticationType`, `adminPassword`, `vmSize`) instead of comments let's use [Bicep's @description decorator](https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/parameters#description).

This decorator will provide contextual help to users in both the Portal and the Azure CLI `az deployment` sub commands who may never see the source.

instead of:

```bicep
// SSH Key or password for the Virtual Machine. SSH key is recommended.
param adminPassword string
```

let's use the decorator like:

```bicep
@description('SSH key or password for the Virtual Machine. SSH key is recommended.')
param adminPassword string
```